index.html

```diff
@@ -1668,111 +1679,110 @@ <h3>Default Value Definitions</h3>
-                  </td>
-                  <td><code>header</code></td>
-              </tr>
-              <tr class="rfc2119-default-assertion" id="td-default-in-2">
-                  <td><code>APIKeySecurityScheme</code></td>
-                  <td>
-                      <code>in</code>
-                  </td>
-                  <td><code>query</code></td>
-              </tr>
-              <tr class="rfc2119-default-assertion" id="td-default-qop">
-                  <td><code>DigestSecurityScheme</code></td>
-                  <td>
-                      <code>qop</code>
-                  </td>
-                  <td><code>auth</code></td>
-              </tr>
-              <tr class="rfc2119-default-assertion" id="td-default-alg">
-                  <td><code>BearerSecurityScheme</code></br>
-                    <code>PoPSecurityScheme</code></td>
-                  <td>
-                      <code>alg</code>
-                  </td>
-                  <td><code>ES256</code></td>
-              </tr>
-              <tr class="rfc2119-default-assertion" id="td-default-format">
-                  <td><code>BearerSecurityScheme</code></br>
-                    <code>PoPSecurityScheme</code></td>
-                  <td>
-                      <code>format</code>
-                  </td>
-                  <td><code>jwt</code></td>
-              </tr>
-              <tr class="rfc2119-default-assertion" id="td-default-flow">
-                  <td><code>OAuth2SecurityScheme</code></td>
-                  <td>
-                      <code>flow</code>
-                  </td>
-                  <td><code>implicit</code></td>
-              </tr>
-          </tbody>
-      </table>
+        <thead>
+            <tr>
+                <th><a>Class</a></th>
+                <th><a>Vocabulary term</a></th>
+                <th>Default value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr class="rfc2119-default-assertion" id="td-default-contentType">
+                <td><code>Forms</code></td>
+                <td>
+                    <code>contentType</code>
+                </td>
+                <td><code>application/json</code></td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-safe">
+                <td><code>ActionAffordance</code></td>
+                <td>
+                    <code>safe</code>
+                </td>
+                <td><code>false</code></td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-idempotent">
+                <td><code>ActionAffordance</code></td>
+                <td>
+                    <code>idempotent</code>
+                </td>
+                <td><code>false</code></td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-op-properties">
+                <td><code>PropertyAffordance</code></td>
+                <td>
+                    <code>op</code>
```

Comment:
Does not work for "Class". Did work when it was context, as op is not part of *Affordance...
Seems to be a big with the change from context.

Reply:
agree. should be changed back to Context